source/java/spring-boot-web/tutorial.html

```diff
@@ -0,0 +1,845 @@
+    <span class="o">}</span>
+<span class="o">}</span>
+</pre></div>
+</div>
+<p><code class="docutils literal"><span class="pre">HelloController.java</span></code> is a little more interesting:</p>
+<div class="highlight-java"><table class="highlighttable"><tr><td class="linenos"><div class="linenodiv"><pre> 1
+ 2
+ 3
+ 4
+ 5
+ 6
+ 7
+ 8
+ 9
+10
+11
+12</pre></div></td><td class="code"><div class="highlight"><pre><span class="nd">@RestController</span>
+<span class="kd">public</span> <span class="kd">class</span> <span class="nc">HelloController</span> <span class="o">{</span>
+    <span class="nd">@RequestMapping</span><span class="o">(</span><span class="s">&quot;/&quot;</span><span class="o">)</span>
+    <span class="kd">public</span> <span class="n">String</span> <span class="nf">hello</span><span class="o">(</span><span class="n">HttpServletRequest</span> <span class="n">req</span><span class="o">)</span> <span class="o">{</span>
+        <span class="n">String</span> <span class="n">greeting</span> <span class="o">=</span> <span class="s">&quot;World!&quot;</span><span class="o">;</span>
+
+<span class="hll">        <span class="n">Application</span> <span class="n">app</span> <span class="o">=</span> <span class="n">ApplicationResolver</span><span class="o">.</span><span class="na">INSTANCE</span><span class="o">.</span><span class="na">getApplication</span><span class="o">(</span><span class="n">req</span><span class="o">);</span>
+</span><span class="hll">        <span class="k">if</span> <span class="o">(</span><span class="n">app</span> <span class="o">!=</span> <span class="kc">null</span><span class="o">)</span> <span class="o">{</span> <span class="n">greeting</span> <span class="o">=</span> <span class="n">app</span><span class="o">.</span><span class="na">getName</span><span class="o">();</span> <span class="o">}</span>
+</span>
+        <span class="k">return</span> <span class="s">&quot;Hello, &quot;</span> <span class="o">+</span> <span class="n">greeting</span><span class="o">;</span>
+    <span class="o">}</span>
+<span class="o">}</span>
+</pre></div>
+</td></tr></table></div>
+<p>Here we have our first taste of Stormpath in action. On line 7 we are getting hold of the Stormpath application and on
+line 8 we are obtaining its name for display.</p>
+<p>Pretty simple setup, right? In the next section, we&#8217;ll layer in some flow logic based on logged-in state. And then we
+will refine that to make use of all the Spring Security has to offer in making our lives easier.</p>
+</div>
+<div class="section" id="some-access-controls">
+<span id="id4"></span><h2><a class="toc-backref" href="#id19">Some Access Controls</a><a class="headerlink" href="#some-access-controls" title="Permalink to this headline">¶</a></h2>
+<p>I am going to say something a little radical here: Don&#8217;t use the code in this section in real life! &#8220;Why have it at all?&#8221;, you may ask.</p>
```

Comment:
I do no fully understand the connotation of this paragraph... I am not saying it is wrong or anything like that, I just do cannot infer the idea behind these lines...